static/js/stream_edit.js

```diff
@@ -120,16 +120,16 @@ function clear_edit_panel() {
     $(".stream-row.active").removeClass("active");
 }
 
-function get_stream_id(target) {
+exports.get_stream_id = function (target) {
     if (target.constructor !== jQuery) {
         target = $(target);
     }
     const row = target.closest(".stream-row, .subscription_settings");
     return parseInt(row.attr("data-stream-id"), 10);
-}
+};
```

Comment:
If we're exporting this, we should add a comment defining its interface -- what does it do when called if one doesn't have the stream-edit UI open?